### ISSUE TITLE
Disable diff colours

### ISSUE THREAD

**Issue by asellitt** [evgenyrodionov/redux-logger#253]
Hey,

Looking at the diff code, it doesnt seem possible to disable the styling of the diffs (in the same way that you can with the other things that are logged.

Are you planning on implementing this in a future release?

**Comment by evgenyrodionov** [comment]
Yep, I’m thinking about `createDiffLogger` with the same options
